File: latihan3 b/Latihan3 Py/PC.py
from Processor import Processor
from Disk import Disk
from RAM import RAM

class PC():
#kelas yang digunakan untuk mengimplementasikan PC
#atribut PC

    prdata = Processor() #Processor data
    dsdata = Disk() #Disk data
    mdata = RAM()	#RAM data
    totalprice = 0  #total pc price
	
    def __init__(self, prdata = Processor(), dsdata = Disk(), rmdata = RAM(), totalprice = 0):
	#costructor
        self.prdata = prdata
        self.dsdata= dsdata
        self.rmdata = rmdata
        self.totalprice = totalprice
	
	#mengeset nilai processor
    def setprocessor(self, prdata):
        self.prdata = prdata
	
	#mengeset nilai Disk
    def setdisk(self, dsdata):
        self.dsdata = dsdata
	
	#mengeset nilai RAM
    def setram(self, rmdata):
        self.rmdata = rmdata
    
    #mengeset nilai totalprice
    def settotal(self, pprice, dprice, rprice):
        self.totalprice = pprice+dprice+rprice
     
	#mengembalikan nilai processor
    def getprocessor(self):
        return self.prdata
	
	#mengembalikan nilai Disk
    def getdisk(self):
        return self.dsdata

	#mengembalikan nilai RAM
    def getram(self):
        return self.rmdata
        
    #mengembalikan nilai total
    def gettotal(self):
        return self.totalprice
    

	
	

File: latihan3 b/Latihan3 PHP/PC.php
<?php

include "Processor.php";
include "Disk.php";
include "RAM.php";


class PC
{ 
	/*kelas yang digunakan untuk mengimplementasikan PC/
	/*atribut PC*/
		
		private $prdata;		//Processor data
		private $dsdata;		//Disk data
		private $rmdata;		//RAM data
		private $totalprice = 0;//total price pc
		
	
		public function __construct(){
		/*costructor*/
		
			$this->prdata = new Processor();
			$this->dsdata = new Disk;
			$this->rmdata = new RAM;
			$this->totalprice = 0;
			
		}
	
	/*mengeset nilai Processor*/
	public function setprocessor($prdata)
	{
		$this->prdata = $prdata;
	}
	
	/*mengeset nilai Disk*/
	public function setdisk($dsdata)
	{
		$this->dsdata = $dsdata;
	}
	
	/*mengeset nilai RAM*/
	public function setram($rmdata)
	{
		$this->rmdata = $rmdata;
	}
	
	/*mengeset nilai Total Price*/
	public function settotal($pprice, $dprice, $rprice)
	{
		$this->totalprice = $pprice + $dprice + $rprice;
	}
	
	/*mengembalikan nilai Processor*/
	public function getprocessor()
	{
		return $this->prdata;
	}
	
	/*mengembalikan nilai Disk*/
	public function getdisk()
	{
		return $this->dsdata;
	}
	
	/*mengembalikan nilai RAM*/
	public function getram()
	{
		return $this->rmdata;
	}
	
	/*mengembalikan nilai Total Price*/
	public function gettotal()
	{
		return $this->totalprice;
	}
	
	function __destruct(){
		/*desstructor*/
		}
}
?>